src/adapter/http/src/data/verify_handler.rs

```diff
@@ -62,6 +64,23 @@ pub struct Response {
 
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
+/// Verify query commitment
+///
+/// See [commitments documentation](https://docs.kamu.dev/node/commitments/) for details.
+#[utoipa::path(
+    post,
+    path = "/verify",
+    request_body = Request,
+    responses(
+        (status = OK, body = Response),
+        (status = BAD_REQUEST, body = Response),
+    ),
```

Comment:
This isn't very clear.
Since many of our queries can result in an error, where do we describe API errors schema?

Reply:
There is a way to document responses better (along with providing nice examples), but I'd rather leave this out of scope of this PR. I want to integrate the base framework first and polish documentation later.